tests/test_skip.py

```diff
@@ -51,13 +51,12 @@ def test_skip_yes(self):
         prog = ql.Program(prog_name, starmon, num_qubits, 0)
 
         qasm_rdr = ql.cQasmReader(starmon, prog)
-        qasm_str = f"""
-            version 1.0
+        qasm_str = f"""version 1.0
             qubits 7
             .{kernel_name}
             cnot q[2],q[5]
             skip 2
-            { x q[0] | y q[1] }
+            {{ x q[0] | y q[1] }}
```

Comment:
Is the double {} actually needed? My tests were working without these so is there a special reason for these?

Reply:
I changed the string to a 'multi line f-string', which doesn't need an explicit "\n" to end lines, and uses {} to insert variable values (like in '{kernel_name}'). To escape "{" one must use "{{"